processing/src/test/java/org/apache/druid/frame/file/FrameFileHttpResponseHandlerTest.java

```diff
@@ -108,45 +105,11 @@ public void setUp() throws IOException
     handler = new FrameFileHttpResponseHandler(channel);
   }
 
-  @Test
-  public void testNonChunkedResponse() throws Exception
```

Comment:
any reason you removed this test?

Reply:
It's testing the case where all content is in the initial HttpResponse, which is no longer possible.